packages/x-components/src/x-modules/search/components/__tests__/redirection.spec.ts

```diff
@@ -153,7 +153,7 @@ describe('testing Redirection component', () => {
 
     wrapper.vm.$x.on('UserClickedARedirection', true).subscribe(onUserClickedARedirection);
     // The delay 0 runs so fast the we need to force the test to simulate a wait.
-    jest.runTimersToTime(1);
+    jest.advanceTimersByTime(1);
```

Comment:
Because of [this](https://jestjs.io/blog/2021/05/25/jest-27#miscellaneous-breaking-changes)

Reply:
👍🏻